src/pages/visualisation-example/VisualisationExamplePage.tsx

```diff
@@ -1,10 +1,22 @@
 import { Visualisation } from "@lib/main";
+import { NavPageWrapper } from "../nav-page-wrapper/NavPageWrapper";
 
 export const VisualisationExamplePage = (): JSX.Element => {
   return (
-    <Visualisation
-      developmentPlanFilepath={"https://raw.githubusercontent.com/digital-land/local-plans-timetable/main/assets/development-plan.csv"}
-      timetableEventsFilepath={"https://raw.githubusercontent.com/digital-land/local-plans-timetable/main/assets/timetable.csv"}
-    />
+    <NavPageWrapper pageTitle="What a Local Plan timetable created online looks like">
+      <p className="govuk-body">
+        This is an{" "}
+        <strong className={`govuk-!-font-weight-bold`}>example</strong> of what
```

Comment:
Out of interest, do we need this class?

I'm also going to create a ticket for changing the existing `b` tags to `strong` (`strong` is better for accessibility)

Reply:
actually we don't need it. I have removed it 👍

Reply:
I ve used it for the Tag component as <strong/> there wasnt displaying bold text but here seems to work fine. (I think for Tag there was a conflict with the "govuk-tag" className )